scylla/src/transport/iterator.rs

```diff
@@ -217,7 +217,7 @@ where
                             error = %error,
                             "Query failed"
                         );
```

Comment:
:wrench: :thought_balloon:  Here I'm not sure if we need the name change or not. After all, `QueryPager`, as the name suggests, should be mainly used for queries, so we can assume such usage in the naming.
OTOH Session is no longer going to log `Query failed`, but rather `Request failed`.

Reply:
I decided to change it (to be consistent with session.rs).